backend/api/Controllers/UserController.cs

```diff
@@ -1,33 +1,32 @@
-using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
-using api.Helpers.Extensions;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using Pims.Api.Helpers;
-using Pims.Dal.Exceptions;
-using MembershipModel = Pims.Dal.Membership.Models;
 using Pims.Api.Models;
+using Pims.Dal.Exceptions;
 using Pims.Dal.Services;
-using System.Linq;
 using Entity = Pims.Dal.Entities;
 using AutoMapper;
+using Pims.Api.Helpers.Extensions;
+using Keycloak = Pims.Api.Models.Keycloak;
 
 namespace Pims.Api.Controllers
 {
     /// <summary>
     /// UserController class, provides endpoints for managing users.
     /// </summary>
-    [Authorize(Roles = "contributor")]
+    [Authorize]
```

Comment:
was this on porpoise?

Reply:
Left over from the code challenge.  I'll update.